### PR TITLE
pornhat.one

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/119958

### CODE REVIEW

**Review by Alex-302 — CHANGES_REQUESTED** [review]
<details><summary>Ad leftover on mobile/desktop:</summary>

[image](https://user-images.githubusercontent.com/8361299/170577973-cbc36388-57c2-4eb9-85a9-3916a2d2f28d.png)
</details><br/>

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Use wildcard `pornhat.*` as everything is the same on `www.pornhat.com`.

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -307,6 +307,8 @@ freehdinterracialporn.in###video > div.border + div[class]
 freehdinterracialporn.in##.my-ad
 ||moanmyip.com/images/removemyphone.png
 justalternativeto.com##.justa-adlabel
+pornhat.*##.top_spot
+pornhat.*##.bn-new
```
Comment (Yuki2718) [inline EnglishFilter/sections/specific.txt:311]: Does this change something on your side? I see no visual difference.

File: EnglishFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -1919,6 +1919,7 @@ indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div#HP-gpt-pa
 indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div[id^="div-gpt-ad-"])
 !
 newzpanda.com#?#.theiaStickySidebar > .stream-item-widget:has(> .widget-title:contains(Advertisement))
+pornhat.*#?##list_videos_videos > div.bn-title-mobile:contains(Advertising)
```
[the remark this answers is not in the document]
Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt:1922]: Yep, your rule is enough.

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/specific.txt:311]: I don't see the need for `pornhat.*##.bn-new`, but `pornhat.*##.top_spot` is required on my end.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -308,7 +308,7 @@ freehdinterracialporn.in##.my-ad
 ||moanmyip.com/images/removemyphone.png
 justalternativeto.com##.justa-adlabel
 pornhat.*##.top_spot
-pornhat.*##.bn-new
+pornhat.one###list_videos_videos > div.bn-title-mobile
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt]: As @Yuki2718 said, `pornhat.*`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: MobileFilter/sections/specific_web.txt [hunk 4]
```diff
@@ -46,6 +46,7 @@ dailynewsonline.jp,yakuji.co.jp,asahi.com,m.winporn.com,slate.com,sankei.com,gam
 !
 h1g.jp##div[style$="width:300px; height:250px; text-align:center;"]
 lepoint.fr###wrap_pos_top
+pornhat.one###list_videos_videos > div.bn-title-mobile
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:49]: This is required for .com too.

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**